src/modules/fancyzones/FancyZonesLib/FancyZonesWindowProcessing.cpp

```diff
@@ -25,13 +25,22 @@ bool FancyZonesWindowProcessing::IsProcessable(HWND window) noexcept
         return false;
     }
 
-    // popup could be the window we don't want to snap: start menu, notification popup, tray window, etc.
-    // also, popup could be the windows we want to snap disregarding the "allowSnapPopupWindows" setting, e.g. Telegram
     bool isPopup = FancyZonesWindowUtils::IsPopupWindow(window);
     bool hasThickFrame = FancyZonesWindowUtils::HasThickFrame(window);
-    if (isPopup && (!hasThickFrame || !FancyZonesSettings::settings().allowSnapPopupWindows))
+    bool hasMinimizeMaximizeButtons = FancyZonesWindowUtils::HasMinimizeMaximizeButtons(window); 
+    if (isPopup)
     {
-        return false;
+        if (hasThickFrame && hasMinimizeMaximizeButtons)
+        {
+            // popup could be the windows we want to snap disregarding the "allowSnapPopupWindows" setting, e.g. Calculator, Telegram   
+        }
+        else if (!FancyZonesSettings::settings().allowSnapPopupWindows || !hasThickFrame || !hasMinimizeMaximizeButtons)
```

Comment:
Logic here needs to be reviewed.
If `if (hasThickFrame && hasMinimizeMaximizeButtons)` is false , then either `hasThickFrame` or `hasMinimizeMaximizeButtons` has to be false.
So `if (!FancyZonesSettings::settings().allowSnapPopupWindows || !hasThickFrame || !hasMinimizeMaximizeButtons)` necessarily has to be true. So `FancyZonesSettings::settings().allowSnapPopupWindows` is just logically ignored. The Setting is doing nothing with these changes.